src/main/java/frc/robot/subsystems/swerve/SwerveCommands.java

```diff
@@ -54,6 +89,18 @@ public static Command getLockSwerveCommand() {
         return command;
     }
 
+    public static Command getReadySpinSwerveCommand() {
+        Command command = new FunctionalCommand(
+                () -> {},
+                SWERVE::readySpinSwerve,
+                inter -> {},
```

Comment:
why is interpol in the code?
![image](https://github.com/GreenBlitz/Off-Season-2024/assets/139997639/6674d54d-778e-4905-ae1c-0a085a8f8418)